symfony.lock

```diff
@@ -1,4 +1,87 @@
 {
+    "api-platform/core": {
```

Comment:
Please remove the diff for `symfony.lock`. Although a basic variant exists in the repo, it is managed manually and we do not commit updates to it.